crates/sequencer/src/rpc.rs

```diff
@@ -92,5 +96,38 @@ pub(crate) fn create_rpc_module<C: sov_modules_api::Context>(
             },
         }
     })?;
+
+    rpc.register_async_method("eth_depositTransaction", |parameters, ctx| async move {
+        let mut params = parameters.sequence();
+        let deposits: Vec<Vec<u8>> = params.next().unwrap();
```

Comment:
we want to parse something like this

"0x213123123"

I don't think it'll work